app/sections/view-solution-capabilities/template.njk

```diff
@@ -22,29 +23,34 @@
   <h3>Generated HTML</h3>
   <pre><code>
     {{- '
-    <div data-test-id="view-solution-capabilities">
-      <div class="bc-c-title-block">
-        <h3 class="nhsuk-u-margin-bottom-2">Capabilities met</h3>
-      </div>
-      <div class="nhsuk-u-padding-left-3">
-        <div data-test-id="view-question-data-bulletlist">
-          <ul>
+    <div class="nhsuk-width-container" data-test-id="view-solution-capabilities">
+      <div class="nhsuk-grid-row">
+        <div class="bc-c-title-block">
+          <h3 class="nhsuk-u-margin-bottom-2">Capabilities met</h3>
+        </div>
+        <p class="nhsuk-grid-column-two-thirds">
+          description
+        </p>
+        <div class="nhsuk-grid-column-two-thirds">
+          <div data-test-id="view-question-data-bulletlist">

```

Comment:
Same again here (create one div containing the "nhsuk-grid-column-two-thirds" class). That way all content will be the same width & don't need to duplicate the class